### ISSUE TITLE
Use AppMetrics as a heartbeat ?

### ISSUE THREAD

**Issue by aborderon** [AppMetrics/AppMetrics#284]
Hello,

I come here to ask a question because I discovered AppMetrics recently.

I am using a client / server application based on the following **Sslstream** example:
https://docs.microsoft.com/fr-fr/dotnet/api/system.net.security.sslstream?view=netcore-2.1

So I have a **TCP TLS connection** between the server and the client.

I would like to know if it is possible to use  #AppMetrics in the form of a **heartbeat** that allows the server to query the client and receive a response every second for example.

This is to be able to check the health of the TCP TLS connection.

Thank you for your information.

**Comment by aborderon** [comment]
Yes, but I can not find anything to test a TCP TLS connection in the AppMetrics documentation.

App.Metrics.Health.Checks.Network only allows you to ping a host if I am not mistaken.

**Comment by alhardy** [comment]
That's something you can implement yourself though right? e.g.

https://www.app-metrics.io/health-checks/#defining-health-checks
